usage/rest-server/src/test/java/brooklyn/rest/BrooklynRestApiLauncher.java

```diff
@@ -150,6 +152,11 @@ public BrooklynRestApiLauncher withoutJsgui() {
         this.deployJsgui = false;
         return this;
     }
+    
+    public BrooklynRestApiLauncher disableHighAvailability(boolean value) {
```

Comment:
Personal preference for fields/methods to be written in the positive rather than the negative.
We could call the method/field `enableHighAvailability(boolean)`, defaulting to true.

Other such examples include:

* `JcloudsLocation.USE_JCLOUDS_SSH_INIT` rather than `DO_NOT_USE_JCLOUDS_SSH_INIT`.
* `org.testng.annotations.Test.enabled` (rather than disabled)